util/build.gradle.kts

```diff
@@ -23,5 +24,11 @@ kotlin {
                 implementation(libs.coroutines.test)
             }
         }
+        val androidMain by getting {
```

Comment:
are paging3 and WorkManager are needed in util ?

Reply:
oh sorry, nice catch! I copied that, then I added `annotation` library and it's pushed already but somehow I forgot to change it also here 🙈